src/validation/validationSchema.ts

```diff
@@ -209,6 +209,7 @@ export default {
     NestedRoot: {
         Comment: true,
         StartAt: true,
-        States: true
+        States: true,
+        ProcessorConfig: true
```

Comment:
This shouldn't be added here. `ProcessorConfig` is not a valid property for other NestedRoot types